model/src/main/proto/performance_metrics.proto

```diff
@@ -26,45 +25,78 @@ message OppiaMetricLog {
   // The current network type of the device.
   NetworkType network_type = 6;
 
-  // The current activity where the event is getting logged.
-  CurrentActivity current_activity = 7;
+  // The metric log of the event.
+  MetricLog metric_log = 7;
 
-  // Structure of the apk size package.
+  // The current screen where the event is getting logged.
+  CurrentScreen current_screen = 8;
+
+  // Structure of a metric log.
+  message MetricLog {
```

Comment:
Why have a separate message type for this vs. just having the one-of in the top-level event?

Reply:
I preferred creating a separate message type because it becomes easier to pipe, pack and unpack data from the logging site to OppiaLogger (+Controller) and then finally to the utility. Just like it has been done in case of 'Context' for the analytics event logs. If we have a top level one-of, we'll always have to transport the entire metric log from one place to another and won't be able to safely transport packaged individual values (like in case of network usage) together.

Reply:
I think that's fine, but perhaps this should just be 'LoggableMetric' instead, as it's not a full log but rather a constituent property of a log object.

Reply:
done.